TiendaPagoCodilityTest/Program.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Newtonsoft.Json;
+
+public class Program
+{
+    public static void Main(string[] args)
+    {
+
+        var resStack = Solution("23 DUP 4 POP 5 DUP -");
+
+        Console.Write(JsonConvert.SerializeObject(resStack));
+    }
+
+    private static Stack<int> Solution(string inputOperations)
+    {
+        string[] arrElements = inputOperations.Split(' ');
+
+        Stack<int> stack = new Stack<int>();
+
+        foreach (string element in arrElements)
+        {
+            int isNumeric;
+            if (int.TryParse(element, out isNumeric))
+            {
+                stack.Push(isNumeric);
+            }
+            else
+            {
+                switch (element)
```

Comment:
This switch could be in another function.

Reply:
Right, can we refactor it together?